src/main/java/com/marklogic/mule/extension/Operations.java

```diff
@@ -125,21 +127,23 @@ public PagingProvider<DatabaseClient, Result<InputStream, DocumentAttributes>> s
         @DisplayName("Directory") @Optional @Example("/customerData") String directory,
         @DisplayName("REST Transform") @Optional String restTransform,
         @DisplayName("REST Transform Parameters") @Optional String restTransformParameters,
-        @DisplayName("REST Transform Parameters Delimiter") @Optional @Example(",") String restTransformParametersDelimiter
+        @DisplayName("REST Transform Parameters Delimiter") @Optional @Example(",") String restTransformParametersDelimiter,
+        @DisplayName("Use Consistent Timestamp") @Optional(defaultValue = "True") boolean useConsistentTimestamp
```

Comment:
For better or worse, we use the term "Consistent Snapshot" in our other tools - starting with DMSDK - so use that here as well. I'll add a definition when I do docs later.